content/asp.net/getting-started/views.md

```diff
@@ -1,20 +1,121 @@
+@{
+    Layout = "_Layout";
+    if (User.HasClaim("Role", "Admin"))
+    {
+        Layout = "_AdminLayout";
+    }
+}
+```
+
+Our `_Layout.cshtml` file is much more opinionated. It essentially creates the HTML page structure that our views will use including the containers and styles with which they are decorated. The template has to take on some dependencies on JavaScript and CSS libraries in order for this to be of any use to someone creating a web site, and you will see evidence of Bootstrap and jQuery here. We will leave full exploration of the HTML, CSS and JavaScript incorporated in this page to the reader, but there are a few important lines to point out in `_Layout.cshtml`. First is how the page head information is being set. 
+
+``` csharp
+<title>@ViewData["Title"] - MvcBasics</title>
+```
+
+Here we see how the layout can interact with the view through the use of shared properties. When rendered, the page title will be set to the value of `@ViewData["Title"]`.
+
+Scanning the layout you'll also come across the following line of code:
+``` csharp
+@RenderBody()
+```
+
+The call to `RenderBody()` is where the bulk of a view's output will be rendered. This is the signal to Razor to inject the parts of a view that are not contained in a section.  Sections are set aside and rendered in a call such as `@RenderSection("scripts", required: false)`. Your views then define corresponding sections and put in the requisite code in order for the page to be output as desired. For example, you might find the following in a view to add JavaScript to your page:
+
+``` csharp
+@section scripts{
+    <!-- add your JavaScript sources here -->
+}
+```
+
+## Using the Data Passed to Our Views
+
+Now that we're starting to see how 
+
+In the sample project we have a controller called `PersonController` with an action method on it called `Index`. Index accepts an integer called `id` as a parameter, retrieves the requested person from the service and returns with a call to the `View` helper method, passing in the person that was resolved from the service. 
+
+``` csharp
+public IActionResult Index(int id)
+{
+    var person = _personService.GetById(id);
+    return View(person);
+}
+```
+
+We use the above code with the view we discussed at the start of this document, which we'll revisit now.
+
+```html
+@model MvcBasics.Models.Person
+
+<h1>Person Details</h1>
+<p>@Model.FirstName @Model.LastName</p>
+```
+
+In the first line of our document we instruct the view engine to use the `Person` class from our project. The person is set in the `Model` property for us at runtime by virtue of us having passed an instance of a person to the `View` helper method in the controller.  To access the properties of our `Person` object, we use the `@Model.FirstName` notation.
+
+There is much more to take advantage of in the Razor view engine and further reading is available at the official documentation, linked to in the "Next Steps" section at the end of the document.
```

Comment:
Why not just link it here (and keep it in the Next Steps as well)?
"...and further reading is available at the official [documentation](link)."